build.gradle

```diff
@@ -1,16 +1,24 @@
 plugins {
     id 'java'
 }
+
+group 'org.example'
+version '1.0-SNAPSHOT'
+
 repositories {
     mavenCentral()
 }
-dependencies {
-    testImplementation 'com.codeborne:selenide:7.5.0'
-    testImplementation group: 'org.testng', name: 'testng', version: '7.10.2'
-    testImplementation 'junit:junit:4.13.2'
-    testImplementation 'org.slf4j:slf4j-simple:2.0.7'
 
+dependencies {
+    implementation 'com.codeborne:selenide:6.19.1'
+    implementation 'org.slf4j:slf4j-api:2.0.7'
+    implementation 'ch.qos.logback:logback-classic:1.4.11'
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.9.2'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.9.2'
+    testImplementation 'org.assertj:assertj-core:3.24.2'
+    testImplementation 'org.testng:testng:7.7.0'  // Додаємо TestNG
 }
+
 test {
-    useTestNG()
+    useJUnitPlatform()
```

Comment:
When we started automation, I suggested using testNG.

- useTestNG(): This sets up Gradle to use **TestNG** as the test framework.
- Since we won’t use **JUnit**, these dependencies are **no longer needed**, simplify your setup and reduce unused libraries. - conflicts may occur.


    implementation 'com.codeborne:selenide:VERSION'  
    implementation 'com.codeborne:selenide-testng:VERSION'  // Let's add this as well, extra TestNG-Selenide integration
    testImplementation 'org.testng:testng:VERSION'